### PR TITLE
add qatlib and a placeholder for the Intel SecL packages

### PR DESCRIPTION
**Pull request by nefigtut**

### CODE REVIEW

File: configs/sst_kernel_tps-intel-qat.yaml [hunk 1]
```diff
@@ -5,52 +5,53 @@ data:
   description: Userspace libraries and tools for Intel QAT hardware crypto device
   maintainer: sst_kernel_tps
 
-# packages:
-#  - do not exist in fedora yet
+  packages:
```
Comment (jwboyer) [inline configs/sst_kernel_tps-intel-qat.yaml:8]: This part is unnecessary.  It'll list these packages for architectures where that package doesn't exist.

**Review by jwboyer — CHANGES_REQUESTED** [review]

File: configs/sst_kernel_tps-intel-qat.yaml (hunk 1 above)
Comment (nefigtut) [reply configs/sst_kernel_tps-intel-qat.yaml]: removed the packages: part. qatlib is now listed in the arch_packages:/x86_64: section only.
thanks josh!

Comment (nefigtut) [reply configs/sst_kernel_tps-intel-qat.yaml]: i hope the yaml is now correct. apologies for the mess.

**Comment by nefigtut** [comment]
the requested changes were made, could you please review again?

**Comment by jwboyer** [comment]
Looks better, thanks!

**Pull request merged**